people/zuzana.json

```diff
@@ -0,0 +1,22 @@
+{
+
+  "key": "zuzana",
+
+  "name": "Zuzana Červínová",
+
+  "nick": "Zuzana",
+
+  "email": "zuzana.cervinova@mild.blue",
+
+  "title_cs": "JUNIOR FRONTEND DEVELOPER",
+
+  "title_en": "JUNIOR FRONTEND DEVELOPER",
+
+  "short_desc_en": "Zuzana is a Bachelor of Arts and a graduate of the Czechitas Digital Academy: Web. At Mild Blue she works as a junior frontend developer.",
+
+  "short_desc_de": "Zuzana ist Bachelor of Arts und Absolventin der Czechitas Digital Academy: Web. Bei Mild Blue arbeitet sie als Junior Frontend Developer.
+  ",
+
+  "short_desc_cs": "Zuzana je bakalář umění a absolvent kurzu Digitální akademie: Web v Czechitas. Ve společnosti Mild Blue pracuje jako junior frontend developer."
```

Comment:
a nechces to trochu rozvest? Treba neco jako.
Zuzana vystudovala to a to, ale po case zjistila, ze ji vlastne bavi programovat a delat hezke weby. Tak se prihlasila do kurzu, naucila se spoustu veci (takovych a makovych) a ted prichazi cas pretavit teorii v praxi. 
Jakoze mi prijde fajn, kdyz jsou ty medailonky takove vic osobni